tests/wpt/metadata/html/dom/interfaces.html.ini

```diff
@@ -8658,6 +8658,12 @@
   [Location interface: window.location must inherit property "reload" with the proper type (2)]
     expected: FAIL
 
+  [Location interface: existence and properties of interface prototype object]
+    expected: FAIL
+
+  [Stringification of window.location]
+    expected: FAIL
```

Comment:
This seems wrong...

Though it's probably due to lack of support for some other part of the test.